documentation/CHANGES.rst

```diff
@@ -440,7 +440,9 @@ Known Issues and Limitations
 - ``std::array`` member function swap cannot be used in DPC++ kernels on Windows platform.
 - ``std::swap`` for ``std::array`` cannot work in DPC++ kernels on Windows platform.
 - Not all functions in <cmath> are supported currently, please refer to `DPC++ library guide <https://www.intel.com/content/www/us/en/docs/onedpl/developer-guide/current/overview.html>`_ for detail list.
-- Due to specifics of Microsoft Visual C++ implementation, some standard math functions for float (including: ``std::ldexp``, ``std::frexp``, ``std::sqrt(std::complex<float>)``) require device support for double precision.
+- Due to specifics of Microsoft Visual C++ implementation, some standard math functions for float (including: ``std::ldexp``, ``std::frexp``, ``std::acosh(std::complex<float>)``,``std::asin(std::complex<float>)``,
+``std::asinh(std::complex<float>)``, ``std::asoc(std::complex<float>)``, ``std::log(std::complex<float>)``, ``std::log10(std::complex<float>)``, ``std::pow(std::complex<float>, std::complex<float>)``,
+``std::sqrt(std::complex<float>)``) require device support for double precision.
```

Comment:
It seems like old release notes (2021.1-beta07) is not the appropriate place to document this.  Perhaps "Known Limitations" within `documentation/library_guide/introduction.rst`

Reply:
Done